pkgs/misc/vscode-extensions/default.nix

```diff
@@ -702,9 +702,15 @@ let
 
       ms-vscode-remote.remote-ssh = callPackage ./remote-ssh {};
 
-      ms-python.python = callPackage ./python {
-        extractNuGet = callPackage ./python/extract-nuget.nix { };
-      };
+      ms-python.python = let
+        raw-package = callPackage ./python {
+          extractNuGet = callPackage ./python/extract-nuget.nix { };
+        };
+      in
+        buildEnv {
+          name = "vscode-extension-ms-python-python-full";
+          paths = [ raw-package self.ms-toolsai.jupyter ];
+        };
```

Comment:
I am not sure about this change and I couldn't find a similar example in the current code. @jonringer does this work? Does this work with buildFHSUserEnv?

Reply:
I would rather avoid bringing in `buildFHSUserEnv` as it's a big hammer.

I think buildEnv is not a bad idea. The more "proper" solution would probably be to have some way to express other extensions as a dependency, then collapse them all when manifesting the final vscode installation.

Reply:
I used `buildEnv` because that's what the `with-extensions.nix` approach is to aggregate the extensions together for use.  It would be nicer if there could simply be a declarative dependency in the `ms-python.python` extension, but I couldn't work out how to cause `with-extensions` to then aggregate the extension without changing it to use something other than its direct use of `buildEnv`.